openml/config.py

```diff
@@ -140,8 +188,12 @@ def set_cache_directory(cachedir):
     cache_directory = cachedir
 
 
+start_use_example_configuration = ConfigurationForExamples.start_use_example_configuration
```

Comment:
Sorry for not being specific enough, could you please also change the name of these functions to make their use clearer?